xarray/backends/netCDF4_.py

```diff
@@ -174,6 +177,12 @@ def _extract_nc4_variable_encoding(variable, raise_on_invalid=False,
 
     return encoding
 
+def _close_ds(ds):
```

Comment:
This is defined redundantly with the function in `h5netcdf_`. But it's probably simpler just to inline it here, too -- no need to get too hung up over repeating a few lines of code.